testlab/node/opencensuscollector/opencensuscollector.go

```diff
@@ -0,0 +1,91 @@
+package opencensuscollector
```

Comment:
nah you’re spot on! the reason i implemented the prometheus collector was because i realized this wasn’t as useful. i think i’ll remove it, but kept as i’m still WIP. prometheus one is working!